src/GraphQL.Tests/Validation/DefaultValuesOfCorrectTypeTests.cs

```diff
@@ -68,9 +68,6 @@ public void variables_with_invalid_default_values()
             _.Error(BadValueForDefaultArgMessage("a", "Int", "\"one\"", "Expected type 'Int', found \"one\"."), 2, 13);
             _.Error(BadValueForDefaultArgMessage("b", "String", "4", "Expected type 'String', found 4."), 3, 16);
             _.Error(BadValueForDefaultArgMessage("c", "ComplexInput", "\"notverycomplex\"", "Expected 'ComplexInput', found not an object."), 4, 22);
-            _.Error("Variable '$a' is invalid. Error coercing default value.", 2, 3);
-            _.Error("Variable '$b' is invalid. Error coercing default value.", 3, 3);
-            _.Error("Variable '$c' is invalid. Error coercing default value.", 4, 3);
```

Comment:
Also notice that we do not have redundant error messages as variables/literals do not attempt to be read if the document is invalid